examples/snap-plugin-collector-rand-streaming/main.go

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 package main
 
 import (
-	"github.com/intelsdi-x/snap-plugin-lib-go/examples/streaming/rand"
+	"github.com/intelsdi-x/snap-plugin-lib-go/examples/snap-plugin-collector-rand-streaming/rand"
```

Comment:
Based on that, some changes are needed in links in README.md - for example here: https://github.com/intelsdi-x/snap-plugin-lib-go/blob/master/examples/streaming/README.md#ready-to-share:
![image](https://user-images.githubusercontent.com/11335874/27227217-e1cdd6aa-52a2-11e7-964e-2c588b64b351.png)